annotations/2.0/apidocs/jakarta.annotation/jakarta/annotation/ManagedBean.html

```diff
@@ -0,0 +1,268 @@
+<a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/annotation/Retention.html?is-external=true" title="class or interface in java.lang.annotation" class="externalLink">@Retention</a>(<a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/annotation/RetentionPolicy.html?is-external=true#RUNTIME" title="class or interface in java.lang.annotation" class="externalLink">RUNTIME</a>)
+public @interface <span class="memberNameLabel">ManagedBean</span></pre>
+<div class="block">The <code>ManagedBean</code> annotation marks a POJO (Plain Old Java Object)
+ as a ManagedBean.  A ManagedBean supports a small set of basic services
+ such as resource injection, lifecycle callbacks and interceptors.</div>
+<dl>
+<dt><span class="simpleTagLabel">Since:</span></dt>
+<dd>Common Annotations 1.1</dd>
+</dl>
+</li>
+</ul>
+</div>
+<div class="summary">
+<ul class="blockList">
+<li class="blockList">
+<!-- =========== ANNOTATION TYPE OPTIONAL MEMBER SUMMARY =========== -->
+<section role="region">
+<ul class="blockList">
+<li class="blockList"><a id="annotation.type.optional.element.summary">
+<!--   -->
+</a>
+<h3>Optional Element Summary</h3>
+<table class="memberSummary">
+<caption><span>Optional Elements</span><span class="tabEnd">&nbsp;</span></caption>
+<tr>
+<th class="colFirst" scope="col">Modifier and Type</th>
+<th class="colSecond" scope="col">Optional Element</th>
+<th class="colLast" scope="col">Description</th>
+</tr>
+<tr class="altColor">
+<td class="colFirst"><code><a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html?is-external=true" title="class or interface in java.lang" class="externalLink">String</a></code></td>
+<th class="colSecond" scope="row"><code><span class="memberNameLink"><a href="#value()">value</a></span></code></th>
+<td class="colLast">
+<div class="block">The name of the Managed Bean.</div>
+</td>
+</tr>
+</table>
+</li>
+</ul>
+</section>
+</li>
+</ul>
+</div>
+<div class="details">
+<ul class="blockList">
+<li class="blockList">
+<!-- ============ ANNOTATION TYPE MEMBER DETAIL =========== -->
+<section role="region">
+<ul class="blockList">
+<li class="blockList"><a id="annotation.type.element.detail">
+<!--   -->
+</a>
+<h3>Element Detail</h3>
+<a id="value()">
+<!--   -->
+</a>
+<ul class="blockListLast">
+<li class="blockList">
+<h4>value</h4>
+<pre><a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html?is-external=true" title="class or interface in java.lang" class="externalLink">String</a>&nbsp;value</pre>
+<div class="block">The name of the Managed Bean. Managed Bean names must be unique within a
+ Jakarta EE module. For each named Managed Bean, Java EE containers must make
```

Comment:
Is this a valid _Java EE_ reference?
Shouldn't it be `Jakarta EE containers`?

Reply:
@m0mus Let's clean this up at the same time as the other updates...  Thanks!

Reply:
this is done too